content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+यह नोड पर चलना चाहिए, भले ही यह कार्यभार अनुप्रयोगों से निकाला जा रहा हो।
+{{< /नोट >}}
+
+## नोड स्थिति
+
+एक नोड की स्थिति में निम्नलिखित जानकारी होती है:
+
+* [पते](#पते)
+* [शर्तें](#शर्त)
+* [क्षमता और आवंटन योग्य](#क्षमता)
+* [जानकारी](#जानकारी)
+
+नोड की स्थिति और अन्य विवरण देखने के लिए आप `kubectl` का उपयोग कर सकते हैं:
+
+```shell
+kubectl describe node <insert-node-name-here>
+```
+
+आउटपुट का प्रत्येक खंड नीचे वर्णित है।
+
+### पते
+
+इन क्षेत्रों का उपयोग आपके क्लाउड प्रदाता या नंगे धातु विन्यास के आधार पर भिन्न होता है।
+
+* होस्टनाम: नोड के कर्नेल द्वारा रिपोर्ट किया गया होस्टनाम। क्यूबलेट के माध्यम से ओवरराइड किया जा सकता है
+  `--hostname-override` पैरामीटर।
+* एक्सटर्नलआईपी: आमतौर पर नोड का आईपी पता जो बाहरी रूप से निष्क्रिय है (से उपलब्ध है
+  क्लस्टर के बाहर)।
+* आंतरिक आईपी: आम तौर पर नोड का आईपी पता जो केवल क्लस्टर के भीतर रूट करने योग्य होता है।
+
+
+### शर्तें {#स्थिति}
+
+`शर्तें` फ़ील्ड सभी `चल रहे` नोड्स की स्थिति का वर्णन करता है। शर्तों के उदाहरणों में शामिल हैं:
+
+{{< table caption = "Node conditions, and a description of when each condition applies." >}}
+| Node Condition       | Description |
+|----------------------|-------------|
+| `Ready`              | `True` if the node is healthy and ready to accept pods, `False` if the node is not healthy and is not accepting pods, and `Unknown` if the node controller has not heard from the node in the last `node-monitor-grace-period` (default is 40 seconds) |
+| `DiskPressure`       | `True` if pressure exists on the disk size—that is, if the disk capacity is low; otherwise `False` |
+| `MemoryPressure`     | `True` if pressure exists on the node memory—that is, if the node memory is low; otherwise `False` |
+| `PIDPressure`        | `True` if pressure exists on the processes—that is, if there are too many processes on the node; otherwise `False` |
+| `NetworkUnavailable` | `True` if the network for the node is not correctly configured, otherwise `False` |
+{{< /table >}}
+
+{{< नोट >}}
+यदि आप कॉर्डन किए गए नोड के विवरण को प्रिंट करने के लिए कमांड-लाइन टूल का उपयोग करते हैं, तो शर्त में शामिल हैं
+'शेड्यूलिंग अक्षम'। कुबेरनेट्स एपीआई में `शेड्यूलिंग डिसेबल` एक शर्त नहीं है; बजाय,
+कॉर्डन किए गए नोड्स को उनकी कल्पना में अनिर्धारित चिह्नित किया गया है।
+{{< /नोट >}}
+
+Kubernetes API में, नोड की स्थिति को `.status` के भाग के रूप में दर्शाया जाता है
+नोड संसाधन का। उदाहरण के लिए, निम्न JSON संरचना एक स्वस्थ नोड का वर्णन करती है:
+
+```json
+"conditions": [
+  {
+    "type": "Ready",
+    "status": "True",
+    "reason": "KubeletReady",
+    "message": "kubelet is posting ready status",
+    "lastHeartbeatTime": "2019-06-05T18:38:35Z",
+    "lastTransitionTime": "2019-06-05T11:41:27Z"
+  }
+]
+```
+
+यदि तैयार स्थिति की `स्थिति` लंबे समय तक `अज्ञात` या `गलत` बनी रहती है
+`पॉड-एविक्शन-टाइमआउट` की तुलना में (एक तर्क पास किया गया
+{{< glosary_tooltip text="kube-controller-manager" term_id="kube-controller-manager"
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.